sdk/webpubsub/Azure.Messaging.WebPubSub/api/Azure.Messaging.WebPubSub.netstandard2.0.cs

```diff
@@ -27,6 +29,8 @@ public partial class WebPubSubServiceClient
         public virtual System.Threading.Tasks.Task<Azure.Response<bool>> ConnectionExistsAsync(string connectionId, Azure.RequestOptions options = null) { throw null; }
         public virtual System.Uri GenerateClientAccessUri(System.DateTimeOffset expiresAt, string userId = null, params string[] roles) { throw null; }
         public virtual System.Uri GenerateClientAccessUri(System.TimeSpan expiresAfter = default(System.TimeSpan), string userId = null, params string[] roles) { throw null; }
+        public virtual Azure.Response GenerateClientToken(string userId = null, System.Collections.Generic.IEnumerable<string> role = null, int? minutesToExpire = default(int?), Azure.RequestOptions options = null) { throw null; }
```

Comment:
We should probably call the parameter `roles`.  And should we replace `minutesToExpire` with a `TimeSpan?` instead of `int?`?  What is Java doing there?

Reply:
I can't find the Java PR.
Can I use a Swagger transform to get the name change? Or, is my only option to pull the method out to a handwritten class definition?

Reply:
There's no Java PR yet, but we should mirror what we do in `GenerateClientAccessUri` since that's got the same features.  We could change it via README transform, but let's hold off until we get an answer on how to properly implement it since I'm leaning toward hiding this if possible.